draft-ietf-tls-esni.md

```diff
@@ -42,6 +42,12 @@ normative:
   RFC2119:
   RFC7918:
 
+informative:
+  WhatWGURLAddressParser:
```

Comment:
Perhaps s/WhatWGURLAddressParser/WHATWG-IPV4/ to be slightly shorter? Also, while not explicitly stated, the example in https://datatracker.ietf.org/doc/html/rfc7322#section-4.8.6.6 implies we like SHOUTY-CASE-WITH-HYPHENS for these tags.

Reply:
👍 The variant in your PR is cleaner, so we can just use that if we decide to land this.